src/helpers/user/user-helper.js

```diff
@@ -6,11 +6,13 @@ const principalApi = getPrincipalApi();
 const principalStatusApiMap = {
   Active: 'enabled',
   Inactive: 'disabled',
+  All: 'all',
 };
 export async function fetchUsers({ limit, offset = 0, orderBy, filters = {}, inModal, matchCriteria = 'partial' }) {
   const { username, email, status = [] } = filters;
   const sortOrder = orderBy === '-username' ? 'desc' : 'asc';
-  const mappedStatus = status.length === 2 ? 'all' : principalStatusApiMap[status[0]] || 'all';
+  const mappedStatus =
+    typeof status === 'string' ? principalStatusApiMap[status] : status.length === 2 ? 'all' : principalStatusApiMap[status[0]] || 'all';
```

Comment:
nitpick - how about using `principalStatusApiMap.All` rather than hardcoded `all`
it's probably safer